.vscode/settings.json

```diff
@@ -2,6 +2,6 @@
   "editor.formatOnSave": true,
   "editor.defaultFormatter": "esbenp.prettier-vscode",
   "editor.codeActionsOnSave": {
-    "source.fixAll.eslint": true
+    "source.fixAll.eslint": "explicit"
```

Comment:
I can't seem to find any documentation on `"explicit"` vs `true`. Even the docs show a boolean in the example: https://github.com/microsoft/vscode-eslint#settings-options

What's the significance of this change?

Reply:
It's not a change I made manually. It must be something in my VS Code config. I'll revert it.